src/OrleansRuntime/Placement/PreferLocalPlacementDirector.cs

```diff
@@ -15,6 +15,10 @@ internal class PreferLocalPlacementDirector : RandomPlacementDirector
         internal override Task<PlacementResult> 
             OnAddActivation(PlacementStrategy strategy, GrainId grain, IPlacementContext context)
         {
+            // if local silo is not active, revert to random placement
+            if (context.LocalSiloStatus != SiloStatus.Active)
```

Comment:
ah, I thought it was a public interface, since we implement it in some of our VSO tests, but those have access to internals. It's fine then :)